docs/source/cuda.rst

```diff
@@ -134,3 +134,14 @@ Jiterator (beta)
 
     jiterator._create_jit_fn
     jiterator._create_multi_output_jit_fn
+
+Sanitizer (prototype)
```

Comment:
Maybe "Stream sanitizer", to clarify it's not a silver bullet that sanitizes every possible aspect of CUDA?

Reply:
You're right, I've changed it.